testdata/IFC4x1ExampleFiles/official/curve.ifc

```diff
@@ -0,0 +1,37 @@
+ISO-10303-21;
```

Comment:
This is not an "official" file, move it one level up to the IFC4x1ExampleFiles folder.

Reply:
Resolved: afcbbd4